wsgi/odyssy/events/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import get_object_or_404
```

Comment:
Can you make these methods as class static methods? That way, we will just have to call `Event.get_latest_events(3)` and so on to access objects. This will ease usage. What do you think?